src/nativeMain/kotlin/org/hisp/dhis/rules/Environment.native.kt

```diff
@@ -0,0 +1,5 @@
+package org.hisp.dhis.rules
+
+actual fun getEnvironment(): String {
+    return "ANDROIDCLIENT"
```

Comment:
This is not right, the Android client uses the JVM target. Native is for native builds (linux, ios, etc)